src/6_branch.js

```diff
@@ -1936,6 +1926,17 @@ Branch.prototype['disableTracking'] = wrap(callback_params.CALLBACK_ERR, functio
 	done();
 }, /* allowed before init */ true);
 
+Branch.prototype['enableTracking'] = function() {
```

Comment:
let me know if you want me to remove this. I just find `branch.disableTracking(false)` to be annoying

Reply:
I think the addition is okay. On other SDKs we are deprecating `disableTracking` for another set of APIs, not sure if it applies to this sdk but just fyi cc @nsingh-branch

Reply:
The new CPP level methods will deprecate disableTracking, so it may not worth it add it if we plan on deprecating it soon anyways. However, i'm not sure when CPP for Web is planned